docs/examples.rst

```diff
@@ -0,0 +1,59 @@
+*****************************************
+Adding parts processing to an application
+*****************************************
+
+A simple example
+================
+
+To add parts processing to an application, one needs to instantiate the
```

Comment:
This section should probably be imperative (no room for creativity) for reproduceable results.

Reply:
Agree, change wording to use strict values instead of suggesting them.